include/zephyr/bluetooth/cs.h

```diff
@@ -620,6 +620,172 @@ int bt_le_cs_create_config(struct bt_conn *conn, struct bt_le_cs_create_config_p
  */
 int bt_le_cs_remove_config(struct bt_conn *conn, uint8_t config_id);
 
+/** @brief CS Security Enable
+ * 
+ * This commmand is used to start or restart the Channel Sounding Security
+ * Start procedure in the local Controller for the ACL connection identified
+ * in the conn parameter.
+ * 
+ * @note To use this API @kconfig{CONFIG_BT_CHANNEL_SOUNDING} must be set.
+ *
+ * @param conn   Connection Object.
+ *
+ * @return Zero on success or (negative) error code on failure.
+ */
+int bt_le_cs_security_enable(struct bt_conn *conn);
+
+struct bt_le_cs_procedure_enable_param {
+	uint8_t config_id;
+	uint8_t enable;
+}
+
+/** @brief CS Procedure Enable
+ * 
+ *  This command is used to enable or disable the scheduling of CS procedures
+ *  by the local Controller, with the remote device identified in the conn
+ *  parameter.
+ * 
+ * @note To use this API @kconfig{CONFIG_BT_CHANNEL_SOUNDING} must be set.
+ * 
+ * @param conn   Connection Object.
+ * @param params Parameters for the CS Procedure Enable command.
+ *
+ * @return Zero on success or (negative) error code on failure.
+ */
+int bt_le_cs_procedure_enable(struct bt_conn *conn, const struct bt_le_cs_procedure_enable_param *params);
+
+struct bt_le_cs_set_procedure_parameters_param {
+	uint8_t config_id;
+	uint16_t max_procedure_len;
+	uint16_t min_procedure_interval;
+	uint16_t max_procedure_interval;
+	uint16_t max_procedure_count;
+	uint32_t min_subevent_len;
+	uint32_t max_subevent_len;
+	uint8_t tone_antenna_config_selection;
+	uint8_t phy;
+	uint8_t tx_power_delta;
+	uint8_t preferred_peer_antenna;
+	uint8_t snr_control_initiator;
+	uint8_t snr_control_reflector;
+};
+
+/** @brief CS Set Procedure Parameters
+ * 
+ * This command is used to set the parameters for the scheduling of one
+ * or more CS procedures by the local controller.
+ * 
+ * @note To use this API @kconfig{CONFIG_BT_CHANNEL_SOUNDING} must be set.
+ * 
+ * @param conn Connection Object.
+ * @param params Parameters for the CS Set Procedure Parameters command.
+ * 
+ * @return Zero on success or (negative) error code on failure.
+*/
+int bt_le_cs_set_procedure_parameters(struct bt_conn *conn, const struct bt_le_cs_set_procedure_parameters_param *params);
+
+/** @brief CS Set Channel Classification
+ * 
+ * This command is used to update the channel classification based on
+ * its local information.
+ * 
+ * @note To use this API, @kconfig{CONFIG_BT_CHANNEL_SOUNDING} must be set.
+ * 
+ * @param channel_classification Bit fields
+ * 
+ * @return Zero on success or (negative) error code on failure.
+ */
+int bt_le_cs_set_channel_classification(uint8_t channel_classification[10]);
+
+struct bt_le_cs_read_local_supported_capabilities_return {
```

Comment:
So I was thinking we wouldn't need:
- struct bt_le_cs_read_local_supported_capabilities_return
- struct bt_le_cs_write_cached_remote_supported_capabilities_param

and could just reuse struct bt_conn_le_cs_capabilities instead